armi/reactor/tests/test_components.py

```diff
@@ -1812,3 +1815,42 @@ def test_finalizeLoadDBAdjustsTD(self):
         comp.p.theoreticalDensityFrac = tdFrac
         comp.finalizeLoadingFromDB()
         self.assertEqual(comp.material.getTD(), tdFrac)
+
+
+class TestPinQuantities(unittest.TestCase):
+    """Test methods that involve retrieval of pin quantities."""
+
+    def setUp(self):
+        self.r = loadTestReactor()[1]
```

Comment:
These test would be faster if you used a smaller test reactor:

```python
r = loadTestReactor(inputFileName="smallestTestReactor/armiRunSmallest.yaml")[1]
```

Would that work for you? If so, it's more than 100 times faster.

Reply:
@zachmprince this was your contribution so I will defer to you

Reply:
Changed the test slightly to use the one-block reactor.